packages/web-components/package.json

```diff
@@ -86,10 +86,10 @@
     "webpack": "^4.43.0"
   },
   "dependencies": {
-    "@microsoft/fast-colors": "^5.1.0",
-    "@microsoft/fast-element": "^1.6.0",
-    "@microsoft/fast-foundation": "^2.32.0",
-    "@microsoft/fast-web-utilities": "^5.0.0",
+    "@microsoft/fast-colors": "^5.1.5",
```

Comment:
This can stay - ideally it resolves either way, nothing is breaking here but this would trigger a cascade for dependents, yes?